### PR TITLE
Update README with new subgraph schema

### PR DESCRIPTION
**Pull request by manumonti**
**Note: Once #17 is merged, the Development query URL must be updated in README**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -1,162 +1,105 @@
...
+This repository contains Threshold Network subgraph.
 
-The subgraphs are open APIs to query data from networks like Ethereum and IPFS. The data is indexed by https://thegraph.com[*The Graph*] decentralized protocol.
+Subgraphs are open APIs to query data from networks like Ethereum and IPFS. The
+data is indexed by https://thegraph.com[*The Graph*] decentralized protocol.
 
 toc::[]
 
+NOTE: Development query URL:
+https://api.studio.thegraph.com/query/24143/threshold/0.0.5
```
Comment (r-czajkowski) [inline README.adoc]: The link doesn't work.

File: README.adoc [hunk 2]
```diff
@@ -1,162 +1,105 @@
...
+
+- `MinStakeAmount`: minimum amount of tokens allowed to stake in Threshold
+Network.
+
+- `PREOperator`: PRE application stake operator's address and timestamp of
+operator confirmation. If timestamp is null, the operator has been bonded but
+not confirmed.
+
+- `DAOMetric`: metrics about the amount of liquid and staked tokens in DAO.
+
```
Comment (r-czajkowski) [inline README.adoc:35]: Non-blocking just a thought: I think we can move the entity docs to a `schema.graphql` file and the docs will be available in the graph studio, for example:
- https://github.com/livepeer/subgraph/blob/main/schema.graphql#L9
- https://thegraph.com/explorersubgraph?id=FDD65maya4xVfPnCjSgDRBz6UBWKAcmGtgY6BmUueJCg&view=Playground

obraz

**Review by r-czajkowski — CHANGES_REQUESTED** [review]
Let's fix the link to the subgraph studio. Overall looking good 👍

File: README.adoc (hunk 2 above)
Comment (manumonti) [reply README.adoc:35]: Good idea! I've just created an issue for this:

https://github.com/threshold-network/subgraph/issues/25

File: README.adoc (hunk 1 above)
Comment (manumonti) [reply README.adoc]: Done!

**Review by r-czajkowski — APPROVED** [review]

**Pull request merged**